UI/Tw.html

```diff
@@ -0,0 +1,496 @@
+<!DOCTYPE html>
+<html lang="ru">
+  <head>
+    <meta charset="utf-8">
+      <meta name="keywords" content="HTML, BSU training, website">
+    <title>Thinhill</title>
+    <link rel="stylesheet" href="styles.css">
+  </head>
+  <body>
+    <div class="container">
+      <header class="header">
+          <span>
+              <p> Thinhill</p>
+          </span>
+          <div class="hSp1">
+             Hello <br> Nadia!
+           </div>
+          <span>
+              <p><b><u><a href="out.html" target="_blank">Sign out</a></u></b></p>
```

Comment:
Is there any practical need to keep such a complicated structure? If there's only one goal: to style component, then please use styles instead of tag nesting